### DISCUSSION TITLE
The payment service of Github Copilot has a VERY SERIOUS BUG!!!

### DISCUSSION THREAD

**Discussion by strugglebak** [community/community#30423] [Copilot Conversations]
Just **DO NOT UPDATE YOUR [PAYMENT METHOD](https://github.com/settings/billing/payment_information)!!!** Otherwise **YOU WILL BE AUTOMATICALLY DEBITED WITH 100$  IF YOU CHOOSE THE YEARLY BILL!!!**. And I've reported this bug to github support, but I didn't get any response yet.

**Comment by strugglebak** [comment]
And I have unbundled my bank card from Paypal to avoid another automatic debit, thx Github, you guys done well. :)

**Comment by strugglebak** [comment]
![image](https://user-images.githubusercontent.com/17985856/185968464-8ef1769c-3f9e-4574-8cdf-d9932a9cc1a1.png)
I'd like to see how long I have to wait before I get feedback.

**Reply by billythekid** [reply]
I'd suggest only opening a single ticket  for fastest resolution. _Bumping_ isn't a thing and multiple tickets only slows the whole queue. 

If you've raised a ticket the support team will be in touch as soon as they can.

**Reply by strugglebak** [reply]
Thx, I've taken your suggestion

**Reply by chandraawardhana** [reply]
I also experienced the same thing, I've paid copilot for one year $100 and my paypal funds have been deducted but my copilot can't be used because its active period has ended.

**Reply by strugglebak** [reply]
> I also experienced the same thing, I've paid copilot for one year $100 and my paypal funds have been deducted but my copilot can't be used because its active period has ended.

aha, this also another stupid bug, now I'm wonding if this Copilot AI is robbing me.

**Reply by yiyuxin** [reply]
same to you

**Comment by strugglebak** [comment]
What makes me wonder is why I still can receive automatic deductions even though I've already cancelled my subscription of GitHub Copilot

**Comment by FaizaNawaz75** [comment]
i read, no amount will be deducted in free trial and I added my credit card info.. 
Within 5 minutes time, 10$ were deducted from my card. No clue where to report this issue.
